packages/daemon/src/rpc/methods/identity.js

```diff
@@ -116,18 +123,29 @@ export const unlinkEthWallet = {
   },
 }
 
-export const addPeerByKey = {
-  params: IDENTITY_ADD_PEER_BY_KEY_SCHEMA,
+export const addPeerByFeed = {
+  params: IDENTITY_ADD_PEER_BY_FEED_SCHEMA,
+  handler: async (
+    ctx: RequestContext, // eslint-disable-line
```

Comment:
Rather than using `eslint-disable`, you should be able to prefix the argument with `_` so it won't complain if it is unused, as it's configured in our ESLint config.